python/tools/desktop_onboarding.py

```
@@ -0,0 +1,291 @@
+        status="ok" if hv_enabled else "warning",
+        details=" ".join(details_parts),
+        remediation=remediation,
+    )
+
+
+def _check_virtualization_linux() -> CheckResult:
+    """Check virtualization prerequisites for Linux hosts."""
+
+    cpuinfo_flags = ""
+    try:
+        with open("/proc/cpuinfo", "r", encoding="utf-8") as handle:
```

Ruff (0.14.0)

Unnecessary mode argument

Remove mode argument

(UP015)